docs/reference/operations-guide/core.md

```diff
@@ -4625,6 +4729,44 @@ Gets the traits of the current store. [Javadoc](https://gchq.github.io/Gaffer/uk
         [ "QUERY_AGGREGATION", "MATCHED_VERTEX", "TRANSFORMATION", "INGEST_AGGREGATION", "PRE_AGGREGATION_FILTERING", "POST_TRANSFORMATION_FILTERING", "POST_AGGREGATION_FILTERING" ]
         ```
 
+## HasTrait
+
+Checks if a Store has a given trait. [Javadoc](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/store/operation/HasTrait.html)
+
+??? example "Example checking if store has trait"
+    === "Java"
+
+        ``` java
+        final HasTrait operation = new HasTrait.Builder()
+                .currentTraits(false)
```

Comment:
This is optional and while the default is true, the significance of this option should be documented.
[You can find information about this option in the codebase](https://github.com/gchq/Gaffer/blob/3b93f97bcbe2f2b064cd399ce9badd325dee78d9/core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/HasTrait.java#L43).